explorer/app/components/Table/table.tsx

```diff
@@ -84,10 +84,15 @@ export const TableComponent = <T extends object>({
     listStaticLoad,
     paginationState,
     relatedListItems,
-    sortState,
+    sorting,
   } = exploreState;
   const { currentPage, pages, pageSize, rows } = paginationState;
-  const sorting = sortState ? [sortState] : undefined;
+  const onSortingChange = (updater: Updater<ColumnSort[]>): void => {
```

Comment:
`onSortingChange` payload needs to be able to type check `updater` - the `resetSorting` API returns `updater` as a value.